pages/api/viewingPermissions/create.ts

```diff
@@ -0,0 +1,47 @@
+import { PrismaClient } from "@prisma/client";
+import { NextApiRequest, NextApiResponse } from "next";
+import Joi from "joi";
+
+const prisma = new PrismaClient();
+type viewingPermissionDTO = {
```

Comment:
Same nit here about type alias names. Thanks for normalizing the fields with camelCase 😄

Reply:
Should we change the file name `viewingPermissions` to `ViewingPermissions` too?

Reply:
I think that's fine. `/api/viewingPermissions` as an HTTP route seems appropriate to me.